FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift

```diff
@@ -37,7 +37,7 @@ class DownloadHandlers {
 
 /// Manager to handle model downloading device and storing downloaded model info to persistent storage.
 class ModelDownloadTask: NSObject {
-  let app: FirebaseApp
+  private let app: FirebaseApp
```

Comment:
Nit for a future PR (doesn't have to be handled now): looks like the only thing we use here is `app.name` so it's likely easier for testing purposes and everything to just pass in `appName: String` instead.

Reply:
Makes sense. I made this change in a couple of other classes as well.